extensions/amp-auto-ads/0.1/attributes.js

```diff
@@ -15,6 +15,7 @@ const NON_DATA_ATTRIBUTE_ALLOWLIST = {
   'height': true,
   'width': true,
   'sticky': true,
+  'json': true,
```

Comment:
Why do you need to do all these? I thought you wanted to access `data-json`, not `json` attribute, right?

Reply:
Okay, so you want to read amp-ad json attribute. Gotcha